scripts/process_extraction_schemas_json

```diff
@@ -1,3 +1,12 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Mar 24 22:37:32 2022
+
+@author: CCloveHH
```

Comment:
It's fine to have authors, but perhaps you want to replace CCloveHH with your Github username? Probably good to add mine as well.